Improve naming consistency

TheRealHaoLiu opened the pull request:
##### SUMMARY
- Rename launch script for launch awx web
- Rename supervisor.conf.j2 to be supervisor_web.conf.j2
- Rename SUPERVISOR_WEB_CONFIG_PATH to SUPERVISOR_CONFIG_PATH

see individual commit for detail explanation

corresponding operator PR https://github.com/ansible/awx-operator/pull/1312


##### ISSUE TYPE
 - Bug, Docs Fix or other nominal change

##### COMPONENT NAME
 - Other

##### AWX VERSION
<!--- Paste verbatim output from `make VERSION` between quotes below -->
```
awx: 21.14.1.dev123+g51c654d222
```


##### ADDITIONAL INFORMATION
<!---
Include additional information to help people understand the change here.
For bugs that don't have a linked bug report, a step-by-step reproduction
of the problem is helpful.
  -->

<!--- Paste verbatim command output below, e.g. before and after your change -->
```

```

TheRealHaoLiu committed:
Rename supervisor.conf.j2 to be descriptive

supervisor.conf.j2 file is the template for supervisor.conf file for the web container rename to supervisor_web.conf make it more clear that it is use for the web container

TheRealHaoLiu committed:
Rename launch script for launch awx web

launch_awx.sh that this PR rename is also now only use for launching awx web container renaming to reflect it's purpose

also remove the no longer needed creation of rsyslog conf as rsyslog is no longer in the web container

Update Dockerfile.j2

relrod reviewed (approved):
Looks fine to me. Thanks for the added consistency.

TheRealHaoLiu commented:
corresponding operator PR https://github.com/ansible/awx-operator/pull/1312

TheRealHaoLiu committed:
rename SUPERVISOR_CONFIG_PATH

previously this is used so that task running in the task container can reach into the web container to restart rsyslog

now that the web container and task container are split there's no longer a way to do that so i renamed this env var to reference where it will now do

which is pointing to the supervisor conf file of the current running container
